src/baldr/tz_alt.cpp

```diff
@@ -526,7 +533,7 @@ native_to_standard_timezone_name(const std::string& native_tz_name,
 }
 
 // Parse this XML file:
-// http://unicode.org/repos/cldr/trunk/common/supplemental/windowsZones.xml
+// https://raw.githubusercontent.com/unicode-org/cldr/master/common/supplemental/windowsZones.xml
```

Comment:
Is the build fetching this at build-time?

Reply:
no. all of the strings needed for all of the target platforms crammed into headers and read directly from memory. thats what my previous comment on this file was talking about. although tbh im not sure that windows has the same indexing scheme as non windows platforms (and if not (i think its not) its a long standing bug that we'll have to tackle in another)